OpenShift/README.md

```diff
@@ -1,38 +1,17 @@
-
-  Once you choose which version of Kubeflow to deploy, run
-
-  ```shell
-  export KFDEF_DIR=<path_to_kfdef>
-  mkdir -p ${KFDEF_DIR}
-  cd ${KFDEF_DIR}
-  export CONFIG_URI=https://raw.githubusercontent.com/IBM/KubeflowDojo/master/OpenShift/manifests/v1.1.0/kfctl_tekton_openshift_minimal.v1.1.0.yaml
-  kfctl apply -V -f ${CONFIG_URI}
-  ```
-
-  Change the `CONFIG_URI` value if choosing the other KfDef manifest.
-
-  Refer to documents in each sub directory of [manifests](manifests) for more detailed install instructions.
+  For users who rather want to run pipelines with [Tekton pipelines](https://github.com/kubeflow/kfp-tekton), a KfDef configuration file is provided in [manifests](manifests) directory. Users should follow the install instructions in this [link](manifests/README.md) to deploy Kubeflow.
 
 * Deploy with the Kubeflow Operator
 
-Kubeflow can also be deployed through the Kubeflow Operator. We are working on adding the Kubeflow Operator to the Operators catalog and to be seen from the OpenShift Container Platform's Web Console. Before that is done, please follow the [document](operator/README.md) in [operator](operator) directory to manually add the operator to the catalog and then deploy Kubeflow.
+  Kubeflow can also be deployed through the Kubeflow Operator. There are also two approaches to install the Kubeflow Operator and deploy Kubeflow, either install manually with command lines or install through OpenShift web console.
+  
+  ***Approach #1:*** Users can follow the [Installing the Kubeflow Operator with kustomize and kubectl](https://www.kubeflow.org/docs/operator/install-operator/#2-installing-the-kubeflow-operator-with-kustomize-and-kubectl) guide on Kubeflow document website to install the operator manually. Once the operator is installed, follow the [Install Kubeflow](https://www.kubeflow.org/docs/operator/install-kubeflow/) guide to deploy Kubeflow. Users can set `KFDEF_URL` to either `https://raw.githubusercontent.com/opendatahub-io/manifests/v1.0-branch-openshift/kfdef/kfctl_openshift.yaml` for Kubeflow install with Kubeflow Pipelines or `https://raw.githubusercontent.com/IBM/KubeflowDojo/master/OpenShift/manifests/kfctl_tekton_openshift_minimal.v1.1.0.yaml` for Kubeflow install with Tekton Pipelines.
+
+  ***Approach #2:*** Since OpenShift Container Platform 4.x, the OpenShift web console integrates with the Operators catalog where users can search for operators and install with several simple clicks. We are working on adding the Kubeflow Operator to the Operators catalog so it can be discovered from the OpenShift Container Platform's Web Console. Before that is done, users can follow the guide in this [link](operator/README.md) to add the Kubeflow Operator to the Operators catalog on the OpenShift cluster and install the operator then deploy the Kubeflow from the OpenShift web console.
```

Comment:
```suggestion
  ***Approach #2:*** Since OpenShift Container Platform 4.x, the OpenShift web console integrates with the Operators catalog where users can search for operators and install with several simple clicks. Users can follow the guide in this [link](operator/README.md) to add the Kubeflow Operator to the Operators catalog on the OpenShift cluster and install the operator then deploy the Kubeflow from the OpenShift web console. 
```

Reply:
Additionally can we point to installing this through ODH Operator?